vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -145,14 +145,7 @@ def parse_pod(pod)
       # TODO, podIP
       containers = pod.spec.containers
       new_result[:container_definitions] = containers.collect do |container_def|
-        parse_container_definition(container_def, pod.metadata.uid)
-      end
-
-      # container instances
-      unless pod.status.nil? || pod.status.containerStatuses.nil?
-        pod.status.containerStatuses.each do |cn|
-          new_result[:containers] << parse_container(cn, pod.metadata.uid)
-        end
+        parse_container_definition(container_def, pod)
```

Comment:
Then here you can do something like:

      new_result[:container_definitions] = containers.collect do |container_def|
        definition = parse_container_definition(container_def, pod.metadata.uid)
        definition[:container] = containers_index[container_def.name]
        definition
      end